code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -58,15 +58,15 @@
 		return
 
 	// initial flight time
-	var/flight_duration =  is_flyby ? DROPSHIP_TRANSIT_DURATION : DROPSHIP_TRANSIT_DURATION * GLOB.ship_alt
+	var/flight_duration =  is_flyby ? DROPSHIP_TRANSIT_DURATION / GLOB.ship_alt : DROPSHIP_TRANSIT_DURATION * GLOB.ship_alt
 	if(optimised)
 		if(is_flyby)
 			flight_duration = DROPSHIP_TRANSIT_DURATION * 1.5
 		else
 			flight_duration = DROPSHIP_TRANSIT_DURATION * SHUTTLE_OPTIMIZE_FACTOR_TRAVEL
```

Comment:
```suggestion
			flight_duration *= 1.5
		else
			flight_duration *= SHUTTLE_OPTIMIZE_FACTOR_TRAVEL
```
This is why you are seeing `GLOB.ship_alt` have no effect because if the ship is optimized; this code completely ignores the existing value of `flight_duration` where you applied `GLOB.ship_alt`